src/interface/src/styles/_media.scss

```diff
@@ -0,0 +1,14 @@
+$breakpoint_lg: 840px;
+
+@mixin on-large() {
```

Comment:
@lastminutediorama ended up adding mixins for this, given that those where the two main media queries we have so far.
Open for suggestions on naming :D

Reply:
I think "large" is fine -- it's clear that this is the max breakpoint we support.